### PR TITLE
docs: copyedit readme

### PR DESCRIPTION
**Pull request by scott-martens**
Ticket: https://github.com/jina-ai/team-tech-content/issues/37

General copyediting and language fixes to the README.md file.

---

- [X] This PR references an open issue
- [ ] I have added a line about this change to CHANGELOG

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,22 +18,27 @@
...
-📈 **Performance promise**: uplift pretrained model and deliver SOTA performance on domain-specific neural search applications.
+📈 **Performance promise**: enhance the performance of pre-trained models and deliver state-of-the-art performance on 
+domain-specific neural search applications.
 
-🔱 **Simple yet powerful**: easy access to 40+ mainstream losses, 10+ optimisers, layer pruning, weights freezing, dimensionality reduction, hard-negative mining, cross-modal model, distributed training. 
+🔱 **Simple yet powerful**: easy access to 40+ mainstream loss functions, 10+ optimisers, layer pruning, weight 
+freezing, dimensionality reduction, hard-negative mining, cross-modal models, and distributed training. 
 
-☁ **All-in-cloud**: instant training with our free GPU; manage runs, experiments and artifacts on Jina AI Cloud without worrying about provisioning resources, integration complexity and infrastructure.
+☁ **All-in-cloud**: train using our free GPU, manage runs, experiments and artifacts on Jina AI Cloud without worrying
```
Comment (gmastrapas) [inline README.md:34]:
```suggestion
☁ **All-in-cloud**: train using our free GPU infrastructure, manage runs, experiments and artifacts on Jina AI Cloud without worrying 
```

File: README.md [hunk 2]
```diff
@@ -18,22 +18,27 @@
...
+many deep learning engineers to do.
 
-Finetuner makes fine-tuning easier, faster and performant by streamlining the workflow and handling all complexity and infrastructure on the cloud.
-With Finetuner, one can easily uplift pre-trained models to be more performant and production ready.
+Jina AI's Finetuner makes fine-tuning easier and faster by streamlining the workflow and handling all complexity and 
+infrastructure in the cloud. With Finetuner, one can easily enhance the performance of pre-trained models, making them 
+production-ready without buying expensive hardware.
 
-📈 **Performance promise**: uplift pretrained model and deliver SOTA performance on domain-specific neural search applications.
+📈 **Performance promise**: enhance the performance of pre-trained models and deliver state-of-the-art performance on
```
Comment (gmastrapas) [inline README.md:28]: we could link the blog no?

**Comment by alexcg1** [comment]
@scott-martens can you change your PR title and git messages to follow [naming conventions](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#-naming-conventions)?

File: README.md [hunk 3]
```diff
@@ -97,37 +102,41 @@ With Finetuner, one can easily uplift pre-trained models to be more performant a
...
 
 ```bash
 pip install -U finetuner
 ```
 
-If you want to encode `docarray.DocumentArray` objects with the `finetuner.encode` function, you need to install `"finetuner[full]"`.
-In this case, some extra dependencies are installed which are necessary to do the inference, e.g., torch, torchvision, and open clip:
+If you want to encode `docarray.DocumentArray` objects with the `finetuner.encode` function, you will need to install 
+`"finetuner[full]"`. This includes a number of additional dependencies, which are necessary for encoding: Torch, 
+Torchvision, and Open Clip:
```
Comment (alexcg1) [inline README.md:119]:
```suggestion
Torchvision, and OpenCLIP:
```

File: README.md [hunk 4]
```diff
@@ -147,9 +156,16 @@ run = finetuner.fit(
     ],
 )
 ```
+This code snippet describes the following steps:
 
-Here, the training data used is gathered from the Jina AI Cloud, however data can also be passed as a CSV file or DocumentArray, as described [here](https://finetuner.jina.ai/walkthrough/create-training-data/).  
-Fine-tuning might take 5 minutes to finish. You can later re-connect your run with:
+* Login to Jina AI Cloud.
+* Select backbone model, training and evaluation data for your evaluation callback.
+* Start the cloud run.
```
Comment (alexcg1) [inline README.md:163]: Number list?

File: README.md [hunk 5]
```diff
@@ -147,9 +156,16 @@ run = finetuner.fit(
     ],
 )
 ```
+This code snippet describes the following steps:
 
-Here, the training data used is gathered from the Jina AI Cloud, however data can also be passed as a CSV file or DocumentArray, as described [here](https://finetuner.jina.ai/walkthrough/create-training-data/).  
-Fine-tuning might take 5 minutes to finish. You can later re-connect your run with:
+* Login to Jina AI Cloud.
```
Comment (alexcg1) [inline README.md:161]: Log in

**Review by gmastrapas — CHANGES_REQUESTED** [review]
Can you also update the CHANGELOG.md file with the changes that this PR brings?

File: README.md (hunk 2 above)
Comment (scott-martens) [reply README.md:28]: Mixed feelings about that. I'm not sure we should commit a link to the blog here. I worry about unstable links. Especially if we're going to recast the README in the future to match the new direction for marketing Finetuner.

File: README.md [hunk 6]
```diff
@@ -18,22 +18,27 @@
...
+📈 **Performance promise**: enhance the performance of pre-trained models and deliver state-of-the-art performance on 
+domain-specific neural search applications.
 
-🔱 **Simple yet powerful**: easy access to 40+ mainstream losses, 10+ optimisers, layer pruning, weights freezing, dimensionality reduction, hard-negative mining, cross-modal model, distributed training. 
+🔱 **Simple yet powerful**: easy access to 40+ mainstream loss functions, 10+ optimisers, layer pruning, weight 
+freezing, dimensionality reduction, hard-negative mining, cross-modal models, and distributed training. 
 
-☁ **All-in-cloud**: instant training with our free GPU; manage runs, experiments and artifacts on Jina AI Cloud without worrying about provisioning resources, integration complexity and infrastructure.
+☁ **All-in-cloud**: train using our free GPU, manage runs, experiments and artifacts on Jina AI Cloud without worrying 
+about resource availability, complex integration, or scalable infrastructure.
```
[the remark this answers is not in the document]
Comment (scott-martens) [reply README.md]: Word duplication for 'availablity'. How does the new wording sound to yo?

File: README.md [hunk 7]
```diff
@@ -18,22 +18,27 @@
 
 <!-- start elevator-pitch -->
 
-Fine-tuning is an effective way to improve the performance on neural search tasks. However, it is non-trivial for many deep learning engineers.
+Fine-tuning is an effective way to improve performance on neural search tasks. However, it can be very difficult for
```
[the remark this answers is not in the document]
Comment (scott-martens) [reply README.md]: I don't. It's a very indirect and computer-geek slang way to say it. How does the new wording sound?

[the remark this answers is not in the document]
Comment (gmastrapas) [reply README.md]: lets go with difficult instead of very difficult?

[the remark this answers is not in the document]
Comment (scott-martens) [reply README.md]: `Fine-tuning is an effective way to improve performance on neural search tasks. However, setting up and performing 
fine-tuning can be very time-consuming and resource-intensive.`

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (scott-martens) [reply README.md]: `...without worrying about resource availability, complex integration, or infrastructure costs.`

**Comment by gmastrapas** [comment]
Screenshot 2022-12-08 at 15 12 26
the font is very small in the bottom

**Comment by scott-martens** [comment]
> Screenshot 2022-12-08 at 15 12 26 the font is very small in the bottom

You're right:

```
<sub><sup>All metrics were evaluated for k@20 after training for 5 epochs using the Adam optimizer with learning rates of 1e-4 for ResNet, 1e-7 for CLIP and 1e-5 for the BERT models.</sup></sub>
```

This guarantees that it'll be awful fine print. Let me see if I can do better

**Comment by scott-martens** [comment]
@gmastrapas Try now.

**Review by gmastrapas — APPROVED** [review]

**Comment by bwanglzu** [comment]
remember to add a line in `CHANGELOG`

**Review by alexcg1 — APPROVED** [review]

**Pull request merged**